app/src/main/java/com/medmapper/v33001/MainViewModel.kt

```diff
@@ -14,42 +15,31 @@ import com.medmapper.v33001.dto.Medicine
 import com.medmapper.v33001.dto.User
 import com.medmapper.v33001.service.IMedicineService
 
-class MainViewModel(var medicineService : IMedicineService) /*= MedicineService())*/ : ViewModel() {
+class MainViewModel(var medicineService : IMedicineService) : ViewModel() {
```

Comment:
Commented out code does not belong in version control, so this is good to remove.